cloud/wfm/examples/index.ts

```diff
@@ -0,0 +1,43 @@
+import * as Promise from 'bluebird';
+import {
+  BaseTask,
+  Executor,
+  ExecutorImpl,
+  ExecutorRepository,
+  Process,
+  ProcessImpl,
+  ProcessInstance,
+  ProcessInstanceImpl,
+  Result,
+  Task,
+  TaskStatus
+} from '../src';
+
+// Derive from BaseTask to implement one that deals with custom business logic
+class MyTask extends BaseTask {
+  public run() {
+    this.status = TaskStatus.ASSIGNED;
```

Comment:
[Antipatter] Can we use setter method. It will allow us to do more than just set status.